Vagrantfile

```diff
@@ -119,6 +119,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     vb.memory = vm_memory
     vb.cpus = vm_num_cpus
   end
+  config.vm.provider "parallels" do |vb, override|
```

Comment:
`vb` stands for VirtualBox. This is not VirtualBox.

Reply:
Thanks! I was not aware of this documentation, This explains a lot. I will do a followup PR for this.